models/issue_comment.go

```diff
@@ -122,9 +122,8 @@ const (
 )
 
 // WithRole enable a specific tag on the RoleDescriptor.
-func (rd RoleDescriptor) WithRole(role RoleDescriptor) RoleDescriptor {
-	rd |= (1 << role)
```

Comment:
I think we need to do is change this line to `return rd | (1 << role)` and remove the next line.